src/views/ObjectView.vue

```diff
@@ -1,30 +1,232 @@
 <script lang="ts" setup>
-import { onMounted } from "vue";
-import { useRoute } from "vue-router";
+import { ref, onMounted } from "vue";
+import { useRoute, RouterLink } from "vue-router";
+import { DataFactory } from "n3";
 import { useUiStore } from "@/stores/ui";
-import PropTableView from "@/views/PropTableView.vue";
+import { useRdfStore } from "@/composables/rdfStore";
+import { useApiRequest } from "@/composables/api";
+import type { Profile } from "@/types";
+import router from "@/router";
+import LoadingMessage from "@/components/LoadingMessage.vue";
+import ErrorMessage from "@/components/ErrorMessage.vue";
+
+const { namedNode, literal } = DataFactory;
 
 const route = useRoute();
 const ui = useUiStore();
+const { store, parseIntoStore, qnameToIri } = useRdfStore();
+const { loading, error, apiGetRequest } = useApiRequest();
+
+const DEFAULT_LABEL_PREDICATES = [qnameToIri("rdfs:label")];
+const DEFAULT_DESC_PREDICATES = [qnameToIri("dcterms:description")];
+
+type objectItem = {
+    iri: string;
+    title?: string;
+    description?: string;
+    links: {
+        parentIri: string;
+        parentTitle?: string;
+        parentLink: string;
+        parentType: {
+            iri: string;
+            title?: string;
+        };
+        link: string;
+        baseClass: {
+            iri: string;
+            title?: string;
+        };
+    }[];
+    types: {
+        iri: string;
+        title?: string;
+    }[];
+};
+
+const defaultProfile = ref<Profile | null>(null);
+const item = ref<objectItem>({} as objectItem);
+const links = ref<string[]>([]);
+
+function getBaseClassFromLink(link: string): {iri: string; title: string} {
```

Comment:
Wouldn't it be better to expose this info in the API response instead of trying to parse the prez link?

Reply:
We can update the profiles to include `rdf:type` values in the response.

Reply:
In discussion: ensure test cases created to ensure class info is included in response.